src/queries/courtConfig.js

```diff
@@ -0,0 +1,40 @@
+import gql from 'graphql-tag'
```

Comment:
I would rename this file to just court, as we are doing with the others queries and inside have the courtConfig query